edx_toggles/toggles/__future__.py

```diff
@@ -1,8 +1,15 @@
 """
-Expose new-style waffle classes. In the future, these imports will be moved to edx_toggles.toggles. Applications that
-want to migrate to the new-style API can do so right now by importing from this module.
+This module had been created to expose new-style waffle classes. These are now available from edx_toggles.toggles.
 """
-# pylint: disable=unused-import
-from .internal.setting_toggle import SettingDictToggle, SettingToggle
```

Comment:
Why were these ever added to `__futures__`?  Let's not add these back in.  We could mentioned this in the changelog if we want to be safe with it, but I'll leave that up to you.

Reply:
It was just convenient to import all classes from a single module. This is not a breaking change, as we now `import *` from `toggles`.